modules/service/src/main/scala/lucuma/odb/Config.scala

```diff
@@ -242,13 +245,19 @@ object Config:
         EmailAddress.from(s).toOption
 
   case class HttpClient(
-    timeout: Duration
-  )
+    timeout: Duration,
+    idleConnectionTime: Duration
+  ):
+    val effectiveIdleConnectionTime: Duration =
+      if idleConnectionTime > timeout
+        then idleConnectionTime
+        else timeout.plus(1.second)
 
   object HttpClient:
     lazy val fromCiris: ConfigValue[Effect, HttpClient] = (
-      envOrProp("HTTP_CLIENT_TIMEOUT").as[Duration].default(45.seconds) // 45s is Ember's default
-    ).map(HttpClient.apply)
+      envOrProp("HTTP_CLIENT_TIMEOUT").as[Duration].default(45.seconds), // 45s is Ember's default
+      envOrProp("HTTP_CLIENT_IDLE_CONNECTION_TIME").as[Duration].default(60.seconds) // 60s is Ember's default
```

Comment:
I wonder if, instead of having both be config parameters, we could just set the idle connection time to be 15 second more than the timeout. That way, whoever sets the config value wouldn't need to know that the idle connection time should be greater than the timeout.

Reply:
That's the behavior (but wit + 1s) via `effectiveIdleConnectionTime` if you don't explicitly set it.